workflows/prognostic_c48_run/runtime/segmented_run/append.py

```diff
@@ -30,6 +32,16 @@ def read_run_config(run_url):
     return fv3config.load(io.BytesIO(s))
 
 
+def print_step_metadata_json(run_url: str, config: dict):
+    info = StepMetadata(
+        job_type="prognostic_run",
+        commit=os.environ.get("COMMIT_SHA"),
+        url=run_url,
+        dependencies={"ml_models": get_model_urls(config)},
+    )
```

Comment:
`args` field is left empty for prognostic run since it typically always has the same `fv3config.yml`, temporary run dir args.